multipaz/src/commonMain/kotlin/org/multipaz/securearea/cloud/CloudSecureArea.kt

```diff
@@ -907,12 +907,14 @@ open class CloudSecureArea protected constructor(
         suspend fun create(
             storage: Storage,
             identifier: String,
-            serverUrl: String
+            serverUrl: String,
+            httpClientEngineFactory: HttpClientEngineFactory<*>
```

Comment:
Need KDoc for this new `httpClientEngineFactory` parameter.

Reply:
Added the missing KDoc in 75566816ec903ba17e558dcc46529d5f6fe15853, amended & force-pushed.